### DISCUSSION TITLE
Make UI placement manual for linking cabinets to fabricators

### DISCUSSION THREAD

**Discussion by StruggleDuck** [FakeFishGames/Barotrauma#7177] [Ideas and suggestions]
instead of automatically placing the UI for cabinets and fabricators when linked, make it so that it has to be manually done in the sub editor. 

(Hi 3e849f2e5c)

**Comment by 3e849f2e5c** [comment]
b

**Reply by StruggleDuck** [reply]
C

**Reply by StruggleDuck** [reply]
best not forget about this

**Reply by StruggleDuck** [reply]
day 12 of dealing with buggy fabricator UI

**Reply by StruggleDuck** [reply]
one day we may get customizable ui placement, but today is not that day

**Reply by 3e849f2e5c** [reply]
This is genuinely annoying, I realistically won't be able to even think about this feature until January. Locking.

**Comment by MaaiG** [comment]
Yep!
